run_experiments.py

```diff
@@ -578,6 +582,11 @@ def main():
                 if not log_project(project, source_dir, args.jobs):
                     continue
             check_project(project, source_dir, config, args.jobs)
+
+            if not config["CodeChecker"]["url"]:
```

Comment:
Does skipping storage imply that we do not want to do the postprocessing? A HTML summarizing the statistics might be useful without CodeChecker results.

Reply:
As of now, I think `post_processing` depends on some values of the CodeChecker server. Although, the point of this is that we don't want to run a server.

In long term, we could refactor this to make HTML summaries available without a CodeChecker server, but for now, I stuck to this approach. Note: the ground-up refactoring is coming, but it will take a while...